python/tskit/trees.py

```diff
@@ -2136,10 +2136,16 @@ def _minlex_postorder_traversal(self, u):
 
     def nodes(self, root=None, order="preorder"):
         """
-        Returns an iterator over the node IDs in this tree. If the root parameter
-        is provided, iterate over the node IDs in the subtree rooted at this
-        node. If this is None, iterate over all node IDs. If the order parameter
-        is provided, iterate over the nodes in required tree traversal order.
+        Returns an iterator over the node IDs reachable from the root(s) in this tree. If
+        the root parameter is provided, only the subtree rooted at this single root node
+        will be iterated over. If this parameter is None, iterate over the node IDs in
```

Comment:
Maybe link to the ``roots()`` method, or somewhere that there's a definition of what a root is?

Reply:
Done